client/src/locales/translations/en_GB.json

```diff
@@ -219,7 +219,8 @@
     "expire": "The post you have been looking for has expired. Please check our Help Board for more such posts.",
     "noActivity": "No activity found",
     "share": "Share",
-    "delete": "Delete",
+    "delete": "Delete post",
```

Comment:
no need for en_GB change, right?

Reply:
yeah only en_US

Reply:
this was probably find and replace `"delete": "Delete",`